### PR TITLE
Improve contract call signing info

### PR DESCRIPTION
**Pull request by CedrikNikita**

### CODE REVIEW

File: src/popup/components/TransactionOverview.vue [hunk 1]
```diff
@@ -234,7 +237,12 @@ export default defineComponent({
       if (innerTx.value.function === TX_FUNCTIONS.claim) {
         transactionOwnerAddress = await decodeClaimTransactionAccount();
       }
-      ownershipAccount.value = getOwnershipAccount(transactionOwnerAddress);
```
Comment (martinkaintas) [inline src/popup/components/TransactionOverview.vue:237]: This change is a bit confusing for me. 
Why is `ownerAddress` not `transactionOwnerAddress`? And why is `ownershipAccount` not just whatever `getOwnershipAccount(transactionOwnerAddress)` returns? 
I suggest to add some comments here or to change the variable names to make this more clear

**Review by martinkaintas — APPROVED** [review]
LGTM in general but I've left a small suggestion

File: src/popup/components/TransactionOverview.vue [hunk 2]
```diff
@@ -27,6 +28,7 @@ import type {
   ITransaction,
   TxFunction,
 } from '@/types';
+import { AeDecodedCallData } from '@/protocols/aeternity/types';
```
Comment (peronczyk) [inline src/popup/components/TransactionOverview.vue:31]: Pleaxe put this line in group with other protocol related imports.

File: src/constants/common.ts [hunk 3]
```diff
@@ -284,6 +284,7 @@ export const APP_LINK_IOS = 'https://apps.apple.com/us/app/superhero-wallet/id15
 
 export const BUG_REPORT_URL = 'https://spgrrc00ymg.typeform.com/to/Kk3Zyjdr';
 export const AGGREGATOR_URL = 'https://superhero.com/';
+export const CHAT_URL = 'https://chat.superhero.com';
```
Comment (peronczyk) [inline src/constants/common.ts:287]:
```suggestion
export const SUPERHERO_CHAT_URL = 'https://chat.superhero.com';
```

File: src/popup/components/Modals/ConfirmTransactionSign.vue [hunk 4]
```diff
@@ -13,14 +13,35 @@
...
+      </div>
       <DetailsItem
         v-if="!!error"
         :label="$t('pages.transactionDetails.reason')"
         :value="error"
         class="reason"
         data-cy="reason"
       />
+      <DetailsItem
+        v-if="decodedCallData && decodedCallData.functionName"
```
Comment (peronczyk) [inline src/popup/components/Modals/ConfirmTransactionSign.vue:41]:
```suggestion
        v-if="decodedCallData?.functionName"
```

File: src/popup/components/Modals/ConfirmTransactionSign.vue [hunk 5]
```diff
@@ -86,6 +107,16 @@
         expandable
         :label="$t('transaction.advancedDetails')"
       >
+        <DetailsItem
+          v-if="decodedCallData && decodedCallData.functionName"
+          :label="$t('modals.confirmTransactionSign.functionName')"
+          :value="decodedCallData.functionName"
```
Comment (peronczyk) [inline src/popup/components/Modals/ConfirmTransactionSign.vue:113]: Function name is displayed here for the 2nd time.

File: src/popup/components/Modals/ConfirmTransactionSign.vue [hunk 6]
```diff
@@ -516,6 +563,19 @@ export default defineComponent({
     height: 56px;
   }
 
+  .subtitle {
+    margin: 8px 0;
+    color: variables.$color-grey-light;
+
+    &.error {
+      color: variables.$color-warning;
```
Comment (peronczyk) [inline src/popup/components/Modals/ConfirmTransactionSign.vue:571]: The class name does not match the color used. I suggest to use `warning` class name.

File: src/popup/components/Modals/ConfirmTransactionSign.vue (hunk 5 above)
Comment (CedrikNikita) [reply src/popup/components/Modals/ConfirmTransactionSign.vue:113]: Yes, that was intended.

**Comment by Liubov-crypto** [comment]
LGTM

**Pull request merged**